sdk/metric/pipeline_registry_test.go

```diff
@@ -699,6 +718,32 @@ func TestIsAggregatorCompatible(t *testing.T) {
 			kind: InstrumentKindHistogram,
 			agg:  AggregationBase2ExponentialHistogram{},
 		},
+		{
+			name: "SyncGauge and Drop",
+			kind: InstrumentKindGauge,
+			agg:  AggregationDrop{},
+		},
+		{
+			name: "SyncGauge and LastValue{}",
```

Comment:
Don't copy naming cruft

```suggestion
			name: "SyncGauge and LastValue",
```